recipe/meta.yaml

```
@@ -12,16 +12,16 @@ source:
 
 build:
   number: 0
-  noarch: python
-  script: {{ PYTHON }} -m pip install . -vv
+  skip: true  # [py<39]
+  script: {{ PYTHON }} -m pip install . -vv  --no-deps --no-build-isolation
 
 requirements:
   host:
-    - python {{ python_min }}
+    - python
     - pdm-backend
     - pip
   run:
-    - python >={{ python_min }}
+    - python
     - anyio >=4.7.0
     - starlette >=0.41.3
```

```suggestion
    - starlette >=0.41.3
  run_constrained:
    - uvicorn >=0.34.0
```

??? https://github.com/sysid/sse-starlette/blob/v2.2.1/pyproject.toml#L34

---

Created ticket to update uvicorn [here](https://anaconda.atlassian.net/browse/PKG-8033)

---

I'll leave that out for now.